src/main/java/com/jfrog/ide/idea/configuration/XrayServerConfigImpl.java

```diff
@@ -169,9 +176,63 @@ void setExcludedPaths(String excludedPaths) {
     @Override
     public ProxyConfig getProxyConfForTargetUrl(String xrayUrl) {
         HttpConfigurable httpConfigurable = HttpConfigurable.getInstance();
-        if (!httpConfigurable.isHttpProxyEnabledForUrl(xrayUrl)) {
+        if (httpConfigurable.USE_PROXY_PAC) {
```

Comment:
Let's add comment, describing how this setting is done by the user (in IDEA), to help the readers of this code better understand it.